test/utils/InstanceTest.sol

```diff
@@ -131,6 +131,10 @@ abstract contract InstanceTest is HyperdriveTest {
         /// @dev The equality tolerance for the short at maturity round trip
         ///      with shares test.
         uint256 roundTripShortMaturityWithSharesTolerance;
+        /// @dev The equality tolerance for `verifyDeposit`.
```

Comment:
After reading this description I'm still not super clear on how to grok the tolerance... like what does 2 mean for the tolerance? 2% tolerance? 2 Wei tolerance? Would be nice to have this info here w/o needing to dive into the functions to find out.